source/default/_patterns/02-molecules/tabs/_tabs.twig

```diff
@@ -0,0 +1,56 @@
+{#
+/**
+ * @file
+ * tabs
+ *
+ * Available variables for tabs:
+ *  - tabs.label [string]: Used for aria-label for tabslist.
+ *  - tabs.show_title_in_panel [boolean]: Shows title in panel.
+ *  - tabs.panel_title_element [string]: h2|(h3)|span Element to use for panel title.
+ *  - tabs.data [array]: array of tab objects.
+ *    - title [string]: Used for Tab title and Header inside of tab panel.
+ *    - panel [string]: Panel content.
+ */
+#}
+{% set seed = random() %}
+{% set title_element = tabs.panel_title_element|default('h3') %}
+<div class="tabs mb-4">
+  <ul class="tabs__list px-1" role="tablist" {{ tabs.label ? "aria-label='#{tabs.label}'" }}>
+    {% for item in tabs.data %}
+      {% set item_uid = "#{seed}-#{loop.index}" %}
```

Comment:
I see you're using string interpolation here. We also have twig concatenation as well:

```twig
{% set item_uid = seed ~ '-' ~ loop.index %}
```

I've err'd on the side of twig concatentation, but I can be convinced to use interpolation instead.

Reply:
I think I was doing interpolation since in our eslinting of js we must use interpolation. Tho the syntax is more finicky than concatenation.